src/TiogaMeshInfo.h

```diff
@@ -24,7 +24,7 @@ struct MeshBlockInfo
     static constexpr int max_vertex_types = 4;
     using GlobalIDType = uint64_t;
 
-    enum QVarType { ROW = 0, COL };
+    enum QVarType : uint8_t { ROW = 0, COL };
```

Comment:
I think `COL` probably should have a value here.

Reply:
I set `COL` to 1.